pushmanager/ui_methods.py

```diff
@@ -4,3 +4,33 @@ def authorized_to_manage_request(_, request, current_user, pushmaster=False):
        (request['watchers'] and current_user in request['watchers'].split(',')):
         return True
     return False
+
+
+def sort_pickmes(_, requests, tags_order):
+    """Sort pickmes based on tags_order
+
+    Args:
+    -          - request handler object
+    requests   - a list of requests
+    tags_order - order to sort requests
+
+    Return: sorted requests list
+    """
+
+    def compare_tags(tags1, tags2):
```

Comment:
Would it be possible to add secondary sorting by username while we're updating this?
Example: If usera has 2 no-verify requests and userb has a no-verify request, the order should always be: usera, usera, userb